src/ralph/api/auth/user.py

```diff
@@ -1,10 +1,24 @@
 """Authenticated user for the Ralph API."""
 
-from typing import Dict, List
+from enum import Enum
+from typing import Dict, List, Literal
 
 from pydantic import BaseModel
 
 
+Scope = Literal[
```

Comment:
Maybe integrate the literal directly in the Pydantic model

Reply:
I left it out for future use (when actually managing scopes)